src/components/TreeMenu/parts/Details/Details.tsx

```diff
@@ -12,12 +12,17 @@ export const Details: FunctionComponent<DetailsProps> = props => {
 
     let content = children;
     if (elementsList) {
-        content = elementsList.map(element => <a className={cx('link')} href={element.path}>{element.title}</a>)
+        content = elementsList.map(element => <a className={cx('link')} href={element.path}><li>{element.title}</li></a>)
     }
 
     return (
         <details>
-            <summary>{title}</summary>
+            <summary>
+                <svg width="17" height="14" viewBox="0 0 17 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Import react-icons library at file begining:
`import { FaFolder } from 'react-icons/fa';`

Use FaFolder icon:
```
return (
        <details>
            <summary>
                <FaFolder/> {title}
            </summary>
            {content}
        </details>
    );
```

Remember to style the icon and add a className